### ISSUE TITLE
[a11y][two_dimensional_scrollables] Tree view is missing semantics role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161690]
https://github.com/flutter/packages/tree/main/packages/two_dimensional_scrollables/lib/src/tree_view

For Tree
| OS | role |
|--------|--------|
| web | tree |
| ios | - |
| macos | NSAccessibilityOutlineRole |
| windows | ROLE_SYSTEM_OUTLINE |
| android | - | 

for Tree item
| OS | role |
|--------|--------|
| web | treeitem |
| ios | - |
| macos | NSAccessibilityRowRole, NSAccessibilityOutlineRowSubrole |
| windows | ROLE_SYSTEM_OUTLINEBUTTON or ROLE_SYSTEM_OUTLINEITEM |
| android | - |
